python/ray/autoscaler/gcp/example-tpu-pod.yaml

```diff
@@ -0,0 +1,49 @@
+# A unique identifier for the head node and workers of this cluster.
+cluster_name: tpupodtest
+
+max_workers: 2
```

Comment:
We have `max_workers=2` here and `min_workers=1` under `ray_tpu`, but since the TPU has two hosts, we actually have reached the `max_workers` bound for this cluster.  Is that right?  And if we had a TPU pod with 4 hosts we'd need `max_workers=4`? It might be worth adding a comment to clarify this

Reply:
Actually in this case the worker refers to the topology, so if we have a v4-16 as our acceleratorType and `max_workers=2` it can create up to 2 v4-16s.

Reply:
Ah makes sense!